MobileCenter/MobileCenter/Internals/Utils/MSUtility+Application.m

```diff
@@ -0,0 +1,27 @@
+#import "MSUtility+ApplicationPrivate.h"
+
+/*
+ * Workaround for exporting symbols from category object files.
+ */
+NSString * MSUtilityApplicationCategory;
+
+@implementation MSUtility (Application)
+
++ (MSApplicationState)applicationState {
+    
+    // App extentions must not access sharedApplication.
+    if (!MS_IS_APP_EXTENSION) {
+        return (MSApplicationState) [[self class] sharedAppState];
+    }
+    return MSApplicationStateUnknown;
+}
+
++ (UIApplicationState)sharedAppState {
+    
+    // Compute selector at runtime for more discretion.
+    SEL sharedAppSel = NSSelectorFromString(@"sharedApplication");
+    return [[((UIApplication *(*)(id, SEL)) [[UIApplication class] methodForSelector:sharedAppSel])(
+                                                                                                    [UIApplication class], sharedAppSel) valueForKey:@"applicationState"] longValue];
```

Comment:
Does your code formater is well setup? This line looks like over the limit of 120 character defined in the `.clang-format` file at the root of the repo.